students/mxwllndrsn/lesson03/list_lab.py

```diff
@@ -0,0 +1,84 @@
+#uw python 210
+#lesson 03
+#max anderson
+
+#list lab
+
+def fruits():
+
+    fruits = ['Apples', 'Pears', 'Oranges', 'Peaches']
```

Comment:
Extra line - can be removed

Reply:
Charles,
This list does not exist globally when ran or imported in iPython. Not sure what the deal is but needs to exist in each function for this to work.

Reply:
Charles was referring to extra line on line 8. (pep8 formatting).

Reply:
Oh, I will be sure to implement this in future. I feel it more readable with space after def but if that's the standard for the course I will use it. (don't think i used it on strformat_lab though)

Do i need to change it to receive grade?